Sprint-2/errors/1.js

```diff
@@ -4,10 +4,62 @@
 // Try playing computer with the example to work out what is going on
 
 function convertToPercentage(decimalNumber) {
-  const decimalNumber = 0.5;
+  //const decimalNumber = 0.5;
   const percentage = `${decimalNumber * 100}%`;
 
   return percentage;
 }
 
-console.log(decimalNumber);
+//let decimalNumber = 0.1;
+
+//console.log(convertToPercentage(decimalNumber));
+console.log(convertToPercentage(0.1));
+
+
+/* My prediction:
+
+There should be an error in line 13, because 
```

Comment:
What kind of error?

Reply:
Frankly speaking, I do not understand the essence of the question. I gave an exhaustive description of what kind of error it is:

There should be an error in line 13, because
the variable decimalNumber is not defined anywhere
and is not declared in the global context.
This variable is a parameter
of the function convertToPercentage(),
which apparently was intended to convert
a decimal number to a percentage. For example:
0.1 is 10%, 0.05 is 5%, and 0.0025 is 0.25%

Please clarify your question, I really do not understand what you are asking me about

Reply:
syntaxError, referenceError etc https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors

But to be honest I only asked because I always try to find SOMETHING to press, extend, or develop, no matter how complete the answer (because high performers should get the same support to go even further), and this was all I could think of! 😂